src/js/tracks/text-track-display.js

```diff
@@ -103,7 +121,20 @@ class TextTrackDisplay extends Component {
       cues.push(track['activeCues'][i]);
     }
 
-    window['WebVTT']['processCues'](window, track['activeCues'], this.el_);
+    if (track['kind'] === 'descriptions') {
+
+      if ((overrides.descriptionsPlayback) && (overrides.descriptionsPlayback === 'screenReaderOnly')) {
```

Comment:
That's technically true, although:
* It's a pretty extreme use case to want captions _and_ descriptions.
* The author of the descriptions ought to avoid them clashing with captions along the timeline, specifically because they want the descriptions to be read out in pauses in the dialog/soundtrack.

Handling this situation in a more sophisticated way will get complicated quickly. What happens if you turn on CC _and_ Subtitles?